cdap-master-spi/src/main/java/io/cdap/cdap/master/spi/environment/MasterEnvironmentRunnableContext.java

```diff
@@ -34,4 +35,12 @@ public interface MasterEnvironmentRunnableContext {
    * Opens a {@link HttpURLConnection} for the given resource path.
    */
   HttpURLConnection openHttpURLConnection(String resource) throws IOException;
+
+
+  /**
+   * Loads a {@link TwillRunnable} instance from a provided class name.
+   *
+   * @return the twill runnable instance
+   */
+   TwillRunnable loadTwillRunnableClass(String className);
```

Comment:
This is not loading a class but rather creating a new instance?

Reply:
Fixed; renamed to `instantiateTwillRunnable`